script.js

```diff
@@ -1,4 +1,117 @@
+// Define current player
+var currentPlayer = 1;
+
+// Function for output
+var myOutputValue = "";
+
+// Define variables to keep track of player's number
+var playerNumber = [];
+var playerOneNum = playerNumber[0];
+var playerTwoNum = playerNumber[1];
```

Comment:
These 2 unused variables can be removed. That said, if you do intend to use them, I suggest not assigning them to `playerNumber[0]` or `playerNumber[1]` on creation / initialisation. 

This is because `playerNumber` has no items as of its creation on line 8. It is initialised to be an empty array in [line 8](https://github.com/rocketacademy/basics-beat-that/blob/e3e941a6585a42407eb2a4218d945119427bf058/script.js#L8), so it means `console.log(playerOneNum)` or `console.log(playerTwoNum)` are going to give you `undefined` for both.

If you want them initialised to store a number, why not start with `0`? `0` does not make sense in the context of the application (dice rolls start from `1` and end with `6`), so at least you have some placeholder number.